lib/dark_worlds_server/bot/bot_client.ex

```diff
@@ -0,0 +1,72 @@
+defmodule DarkWorldsServer.Bot.BotClient do
+  alias DarkWorldsServer.Communication
+  alias DarkWorldsServer.Communication.Proto.GameEvent
+  alias DarkWorldsServer.RunnerSupervisor.BotPlayer
+  use WebSockex, restart: :temporary
+  require Logger
+
+  def start_link(%{game_id: game_id, config: config}) do
+    client_id = UUID.uuid4()
+    ws_url = Application.fetch_env!(:dark_worlds_server, DarkWorldsServer.Bot) |> Keyword.get(:game_server_url)
```

Comment:
question,can't the request itself contain the url of the game server to play games with?

Reply:
idk if that supposes a security issue

Reply:
This is a config thing, this way its not hardcoded and we have an env var to change it